benchexec/tablegenerator/__init__.py

```diff
@@ -962,6 +962,9 @@ def round_to_MHz(value):
                         return value
                 runSetResult.attributes[key] = Util.prettylist(map(round_to_MHz, values))
 
+            elif key == 'host':
+                runSetResult.attributes[key] = Util.prettylist(values, True)
```

Comment:
`True` is pretty non-intuitive here, better have something like `merge_common_values=True`.